crates/gitbutler-core/tests/suite/virtual_branches/create_virtual_branch_from_branch.rs

```diff
@@ -103,7 +103,7 @@ async fn integration() {
     {
         // should mark commits as integrated
         controller
-            .fetch_from_target(project_id, None)
+            .fetch_from_remotes(project_id, None)
```

Comment:
I love this rename, `remote` is a word I understand. Probably `target` for GB ultimately is only that, even though I don't know if there are some (subtle) differences yet.